README.md

```diff
@@ -4,6 +4,26 @@ This action uses aws cli to login to EKS and deploy a helm chart.
 
 ## Inputs
 
+### `atomic`
+
+If set, upgrade process rolls back changes made in case of failed upgrade.
+
+### `wait`
+
+Whether the upgrade should wait for all resources to be Ready before completing.
+
+### `debug`
+
+Whether to enable debug logging.
+
+### `extra-args`
+
+Any extra command line arguments to pass to Helm.
+
+### `aws-secret-access-key`
+
+AWS secret access key part of the aws credentials. This is used to login to EKS.
```

Comment:
It may be just me, but it looks like the documentation for `aws-secret-access-key` is in here twice now (lines 23-25, and then again on lines 27-29).